README.md

```diff
@@ -95,3 +133,36 @@ does *not* support such comments, however.
     }
 }
 </pre>
+
+<pre>
+# a metadata based whitelist
+{
+    "filesystem": {
+        "audits": {
+            "bsc#123456": {
+                "comment": "some typical special files",
+                # here we use a meta entry instead of a "digests" entry to
+                # whitelist file properties in contrast to file contents.
+                "meta": {
+                    "/some/file": {
+                        # denotes the whitelisted type of file. supported
+                        # characters are currently:
+                        # '-': regular file
+                        # 'd': directory
+                        # 'c': character device
+                        # 's': UNIX domain socket
+                        "type": "d",
```

Comment:
as you showcase dev options below I would use s/b here. Also "b" is missing in the list